cypress/e2e/hermioneAccount.cy.js

```diff
@@ -0,0 +1,73 @@
+import { faker } from '@faker-js/faker';
+/// <reference types='cypress' />
+
+describe('Bank app', () => {
+  const accountBalance = 5096;
+  const depositAmount = faker.number.int({ min: 500, max: 1000 });
+  const withdrawAmount = faker.number.int({ min: 50, max: 500 });
+  const balanceAfterDeposit = accountBalance + depositAmount;
+  const balanceAfterWithdraw = balanceAfterDeposit - withdrawAmount;
+  const user = 'Hermoine Granger';
+  const accountNumber = '1001';
+  const changeAccountNumber = 1003;
```

Comment:
The `changeAccountNumber` is defined as a number, whereas `accountNumber` is a string. Ensure consistency in the data type for account numbers to avoid potential issues.